windtunnel/display.py

```diff
@@ -102,8 +103,27 @@ def add_mesh(self,
             opacity=opacity,
             show_edges=show_edges,
             scalars=scalars,
+            clim=clim,
         )
 
+    def add_force_coefficients(self, coefficients: dict):
```

Comment:
Seems a bit unrelated with the title of the Pull request...  Should the title and description be updated (or ideally, add this in separate pull request)?